core/src/scene/scene.h

```diff
@@ -31,6 +31,8 @@ struct Stops;
  * Scene is a singleton containing the styles, lighting, and interactions defining a map scene
  */
 
+using GlobalUpdate = std::pair<std::string, YAML::Node>;
```

Comment:
Yeah I know but the reason I added it because `std::vector<std::pair<std::string, YAML::Node>>` looks ugly. Will remove it though.